redaxo/src/core/lib/console/setup/check.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @author staabm
+ *
+ * @internal
+ */
+class rex_command_setup_check extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Check the commandline interface (CLI) environment for REDAXO requirements')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output)->getErrorStyle();
+        
+        $errors = rex_setup::checkEnvironment();
+        if (count($errors) == 0) {
+            $io->success(rex_i18n::msg('setup_308'));
+        }
+        
+        $res = rex_setup::checkFilesystem();

```

Comment:
Sollte die db connection gechecked werden in diesm command?

Reply:
würde ich sagen, alternativ als warning mit ausgeben

Reply:
aktuell checken wir nur dinge aus dem setup step: servercheck
Db-conn-check wäre der schritt danach, der auch usereingaben erfodert. Ggf sollte das in einem separatem setup commando sein, bin aber noch unschlüssig.

Reply:
Ich glaube, ich würde es ruhig hier mit integrieren.